roles/openshift_master/handlers/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
-- name: restart openshift-master
-  service: name=openshift-master state=restarted
+- name: restart master
+  service: name=master state=restarted
```

Comment:
There may be some confusion here.  We need:

~~~
name: restart master
service: name={{ product_type }}-master state=restarted
~~~

The service is either name `openshift-master` or `atomic-enterprise-master`.  Haven't only name=master won't work.  The same fix is needed for the node.